resources/views/service-template/index.blade.php

```diff
@@ -0,0 +1,164 @@
+@extends('layouts.librenmsv1')
+
+@section('title', __('Services Templates'))
+
+@section('content')
+    <div class="container-fluid">
+        <x-panel id="manage-services-templates-panel">
+            <x-slot name="title">
+                <span class="fa-stack" aria-hidden="true">
+                    <i class="fa fa-square fa-stack-2x"></i>
+                    <i class="fa fa-cogs fa-stack-1x fa-inverse"></i>
+                </span> @lang('Services Templates')
+            </x-slot>
+            <div class="row" style="padding-bottom: 16px;">
+                <div class="col-md-12">
+                    <a type="button" class="btn btn-primary" href="{{ route('services.templates.create') }}">
+                        <i class="fa fa-plus"></i> @lang('New Service Template')
+                    </a>
+                    <button type="button" title="@lang('Apply Service Templates')" class="btn btn-success" aria-label="@lang('Apply Service Templates')"
+                            onclick="discover_st(this, '{{ route('services.templates.discover') }}')">
+                        <i
+                            class="fa fa-refresh" aria-hidden="true"></i> @lang('Apply Service Templates')</button>
+                </div>
+            </div>
+            @foreach($device_groups as $device_group)
+                <x-panel id="manage-services-templates-panel-dg">
+                    <x-slot name="title">
+                        <i class="fa fa-th fa-fw fa-lg" aria-hidden="true"></i> @lang($device_group->name)
+                    </x-slot>
+                    <div class="table-responsive">
+                        <table id="manage-services-templates-table" class="table table-condensed table-hover">
+                            <thead>
+                            <tr>
+                                <th>@lang('Name')</th>
+                                <th>@lang('Check Type')</th>
+                                <th>@lang('Parameters')</th>
+                                <th>@lang('Remote Host')</th>
+                                <th>@lang('Description')</th>
+                                <th>@lang('Modified')</th>
+                                <th>@lang('Ignored')</th>
+                                <th>@lang('Disabled')</th>
+                            </tr>
+                            </thead>
+                            <tbody>
+                                @foreach($service_templates as $service_template)
+                                    @if($service_template->device_group_id == $device_group->id)
```

Comment:
With the changes above;
```suggestion
                                @foreach($device_group->serviceTemplates as $service_template)
```